daemon/container_operations_solaris.go

```diff
@@ -2,22 +2,89 @@
 
 package daemon
 
-import "github.com/docker/docker/container"
+import (
+	"os"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/docker/docker/container"
+	"github.com/docker/docker/pkg/fileutils"
+	"github.com/docker/docker/pkg/mount"
+	"github.com/docker/docker/runconfig"
+	"github.com/docker/libnetwork"
+)
 
 func (daemon *Daemon) setupLinkedContainers(container *container.Container) ([]string, error) {
 	return nil, nil
 }
 
 // getSize returns real size & virtual size
 func (daemon *Daemon) getSize(container *container.Container) (int64, int64) {
-	return 0, 0
+	var (
```

Comment:
It's a copy of the version in `container_operations_unix.go`, could it be moved to a common files to avoid duplications? We'd definitely forget to update the `solaris` version if we ever had to made changes to this routine.